src/components/coffee-input.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import {connect} from 'react-redux';
+
+import {addCoffee} from '../database/coffee';
+import {FancyInput, FancyDropdown, FancyButton} from './input-stuff.js'
+
+
+// I'm using the same name...that feels wrong. Is it going to break it?
+// or just bad behavior?
+const FancyForm = (props) => {
+
+  function handleSubmit(event) {
+
+    event.preventDefault();
+
+    const timeNow = new Date();
+
+    const newCoffee = {
+      dateCreated: timeNow.getTime(),
+      roaster: event.target.roaster.value,
+      dateRoasted: event.target.dateRoasted.value,
+      origin: event.target.origin.value,
+      price: event.target.price.value,
+      quantity: event.target.quantity.value,
+      tags: event.target.tags.value.split(',').map(str => str.trim()),
+      tastingNotes: event.target.notes.value.split(',').map(str => str.trim())
+    };
+
+    if (props.userid) {
+      console.log("coffee added to database");
+      addCoffee(props.userid, newCoffee);
+    }
+  }
+
+  // is "done" something a person can select? Or is it calculated?
```

Comment:
i think we should set `done` to be either `null` or `true` or a timestamp for when you finish some coffee, so i guess the option to set it manually, but default to deriving it?